spell/spell-crafter-goerli-workflow.md

```diff
@@ -74,6 +75,9 @@ PR: https://github.com/makerdao/spells-goerli/pull/TODO
       * [ ] Check IF oracle deployment is required (e.g. univ3-lp-oracle, new ilk pip, ...)
   * [ ] Ensure every spell variable is declared as public/internal
   * [ ] Consider `immutable` visibility when fetching addresses from the `ChainLog` via `DssExecLib.getChangelogAddress` and use `constant` for static addresses
+    * [ ] Fetch addresses as type `address` and wrap with `Like` suffix interfaces inline (when making calls) unless archive patterns permit otherwise (Such as `MKR`)
+    * [ ] Use the [DssExecLib Core Address Helpers](https://github.com/makerdao/dss-exec-lib/blob/master/src/DssExecLib.sol#L166) where possible (e.g. `DssExecLib.vat()`)
+    * [ ] Variable names of addresses fetched from the `ChainLog` match the corresponding ChainLog key (e.g. `MCD_VAT` rather than `vat`) except where archive pattern differs (e.g. MKR)
```

Comment:
Existing:

```* [ ] Use variable names of addresses fetched from the `ChainLog` match the corresponding ChainLog key (e.g. `MCD_VAT` rather than `vat`) except where archive pattern differs (e.g. MKR)```

New:

```If addresses are fetched from the `ChainLog`, the variable name must match the value of the ChainLog key for that address (e.g. `MCD_VAT` rather than `vat`), except where the archive pattern differs from this pattern (e.g. MKR)```

This refers to an action for the crafter and a check for the reviewer, which raises a different question - do we want actions that refer to the same thing to have different wording on the crafter and reviewer checklists or do we want the wording to be the same (as it currently is) at the cost of checklist items sometimes not being as readable in terms of grammar or syntax.

Of course the benefit of the existing method if that maintenance of the checklist is easier and it's easy to spot corresponding checklist items.

I will update this entry but this stuff poses a good question